src/Magento/FunctionalTestingFramework/Console/StaticChecksCommand.php

```diff
@@ -65,32 +80,41 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $this->ioStyle = new SymfonyStyle($input, $output);
         try {
-            $this->validateInputArguments($input, $output);
+            $this->validateInput($input);
         } catch (InvalidArgumentException $e) {
             LoggingUtil::getInstance()->getLogger(StaticChecksCommand::class)->error($e->getMessage());
-            $output->writeln($e->getMessage() . " Please fix input arguments and rerun.");
```

Comment:
Running below command gave me 0 errors message, probably because it's not a test module path. Should an error be thrown instead?
`bin/mftf static-checks deprecatedEntityUsage --path /Users/sunnikri/workspace/magento2ce`

Reply:
Fixed